tools/modules/software/module_netbox.sh

```
@@ -0,0 +1,164 @@
+			-e SUPERUSER_EMAIL="${SUPERUSER_EMAIL}" \
+			-e SUPERUSER_PASSWORD="${SUPERUSER_PASSWORD}" \
+			-e DB_NAME=netbox \
+			-p ${module_options["module_netbox,port"]}:8080 \
+			-v "${NETBOX_BASE}/config:/etc/netbox/config" \
+			-v "${NETBOX_BASE}/reports:/etc/netbox/reports" \
+			-v "${NETBOX_BASE}/scripts:/etc/netbox/scripts" \
+			--restart unless-stopped \
+			netboxcommunity/netbox:latest
+			# waiting for web
+			sleep 5
+			for s in {1..30};do
```

Shellcheck (0.10.0)

[warning] s appears unused. Verify use (or export if used externally).

(SC2034)